src/views/Profile.vue

```diff
@@ -138,19 +136,16 @@
 </template>
 
 <script setup>
-import {  reactive } from "vue";
 import { Disclosure } from "@headlessui/vue";
 import App from "@/layouts/App.vue";
+import { userdata } from "../utilities/GlobalFunctions";
+
 const user = {
```

Comment:
Is this still needed?

Reply:
yes. as place holder for image

Reply:
So why do we need the whole object and not just the url?